helm/charts/hpe-csi-driver/values.schema.json

```diff
@@ -120,7 +120,8 @@
                     "title": "CHAP password",
                     "description": "Between 12 and 16 characters",
                     "type": "string",
-                    "default": ""
+                    "default": "",
+                    "pattern": "^$|^[a-zA-Z0-9+_)(*^%$#@!]{12,16}$"
```

Comment:
The regex parenthesis doesn't seem to be valid.
 If we are looking for alphanumeric only the following regex would suffice
`^$|^[a-zA-Z0-9+_]{12,16}$`

To keep the above regex valid, it needs to be changed to 
`^$|^[(a-zA-Z0-9+_)(*^%$#@!)]{12,16}$` . 
Could you please verify the change

Reply:
I added the regex using this https://github.com/hpe-storage/common-host-libs/blob/master/linux/iscsi.go#L42 as reference.

Reply:
Also, the round braces are just characters and not an expression.

Reply:
Ok. thanks for the clarification. Looks good then